extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/hll/sql/HllSketchBaseSqlAggregator.java

```diff
@@ -211,4 +205,11 @@ protected abstract Aggregation toAggregation(
       boolean finalizeAggregations,
       AggregatorFactory aggregatorFactory
   );
+
+  private boolean validateInputComplexTypeName(ColumnType columnType)
```

Comment:
Since the input to this method is a column type, and not a the complex type name, we should either have it `validateInputType` or `validateInputColumnType`. 
nit:
```suggestion
  private boolean validateInputType(ColumnType columnType)
```